LibTCPServer.emx

```diff
@@ -943,6 +953,13 @@ catch (Poco::Exception& exc) {
           <type xmi:type="uml:PrimitiveType" href="platform:/plugin/com.ibm.xtools.umldt.rt.cpp.core/libraries/CppPrimitiveDatatypes.emx#_f0875RKyEdyaM9S2zG4bgA?CppPrimitiveDatatypes/bool?"/>
         </defaultValue>
       </ownedAttribute>
+      <ownedAttribute xmi:id="_rXaeQPNBEe6D8ogtBjw-1g" name="maxSupportedClient" visibility="public" isStatic="true">
```

Comment:
We must remember to mention this new configuration parameter in the README for the GitHub repo.

Reply:
updated in readme file.